src/metatensor/models/cli/conf/architecture/experimental.soap_bpnn.yaml

```diff
@@ -1,12 +1,9 @@
-# default hyperparameters for the SOAP-BPNN model
 model:
   soap:
     cutoff: 5.0
     max_radial: 8
     max_angular: 6
     atomic_gaussian_width: 0.3
-    radial_basis:
-      Gto: {}
```

Comment:
why removed it can only be GTO?

Reply:
Yes rascaline only supports `Gto` and splined radial basis. For the latter you need a Python class which we can provide via the command line.

I can add a comment about the radial basis!